elyra/airflow/bootstrapper.py

```diff
@@ -328,20 +343,36 @@ def execute(self) -> None:
                 f"executing python script using " f"'python3 {python_script}' to '{python_script_output}'"
             )
             t0 = time.time()
-            with open(python_script_output, "w") as log_file:
-                subprocess.run(["python3", python_script], stdout=log_file, stderr=subprocess.STDOUT, check=True)
+
+            run_args = ["python3", python_script]
+
+            print("Processing file:", python_script)
```

Comment:
Same comment as previous, looks like `logger.info` is probably best given the other `info` calls.